mobile/ios/simulator/ios-emulator-services.ts

```diff
@@ -12,6 +12,11 @@ class IosEmulatorServices implements Mobile.IiOSSimulatorService {
 		return "";
 	}
 
+	public async getRunningEmulatorId(image: string): Promise<string> {
+		//todo: plamen5kov: fix later if necessary
+		return "";
```

Comment:
We can verify this by looking at the .js code, right? Can you please share it so that we all know how it looks?